content/en/news/releases/1.7.x/announcing-1.7.5/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Announcing Istio 1.7.5
+linktitle: 1.7.5
+subtitle: Patch Release
+description: Istio 1.7.5 patch release.
+publishdate: 2020-11-19
+release: 1.7.5
+aliases:
+- /news/announcing-1.7.5
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.7.4 and Istio 1.7.5
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** pilot agent app probe connection leak. ([Issue #27726](https://github.com/istio/istio/issues/27726))
+
+- **Fixed** how `install-cni` applies `istio-cni` plugin configuration, `Install-cni` will now remove existing `istio-cni` plugins from the CNI config before inserting `istio-cni` plugin configuration (new behavior) rather than just append a new configuration to the existing list (former behavior). ([Issue #27771](https://github.com/istio/istio/issues/27771))
```

Comment:
```suggestion
- **Fixed** how `install-cni` applies `istio-cni` plugin configuration, `install-cni` will now remove existing `istio-cni` plugins from the CNI config before inserting `istio-cni` plugin configuration (new behavior) rather than just append a new configuration to the existing list (former behavior). ([Issue #27771](https://github.com/istio/istio/issues/27771))
```

Reply:
Split the sentence after plugin configuration.

Not sure the (new behavior) and (old behavior) are needed.

Also will ~~now~~ remove and rather than ~~just~~ append

Reply:
This might be better: 

```
- **Fixed** how `install-cni` applies `istio-cni` plugin configuration. Previously, new configurations would be appended to the list. This has been changed to remove existing `istio-cni` plugins from the CNI config before inserting new plugins. 
```

Reply:
Done. Thanks!